packages/grafana-e2e-selectors/src/selectors/pages.ts

```diff
@@ -49,6 +49,7 @@ export const Pages = {
         sectionItems: (item: string) => `Dashboard settings section item ${item}`,
         saveDashBoard: 'Dashboard settings aside actions Save button',
         saveAsDashBoard: 'Dashboard settings aside actions Save As button',
+        title: () => 'h3[class=dashboard-settings__header]',
```

Comment:
I think it's better if we add an aria-label to dashboard title instead if possible?

Reply:
Yeah, how do I do that for the angular views?